Releasing.md

```diff
@@ -216,26 +217,14 @@ For the body of the post, just copy this checklist and again replace all occurre
 <h3>Sync the Release to Development Branches</h3>
 <!-- /wp:heading -->
 
-<!-- wp:paragraph -->
-<p>o If there are any conflicts in the gutenberg PR, merge <code>trunk</code> into it and resolve them.</p>
-<!-- /wp:paragraph -->
-
-<!-- wp:paragraph -->
-<p>o In gutenberg-mobile, create a branch from <code>trunk</code> with a name like <code>merge_release_X.XX.X_to_develop</code> and open PR to <code>develop</code>. If there are any merge conflicts, merge <code>develop</code> into the PR and resolve them.</p>
-<!-- /wp:paragraph -->
+ <!-- wp:paragraph -->
+-<p>o If there are any conflicts in the gutenberg PR, merge <code>trunk</code> into it and resolve them.</p>
+-<!-- /wp:paragraph -->
 
 <!-- wp:paragraph -->
 <p>o Merge the gutenberg PR to <code>trunk</code>.</p>
 <!-- /wp:paragraph -->
 
-<!-- wp:paragraph -->
-<p>o Update the <code>gutenberg</code> reference in the gutenberg-mobile <code>merge_release_X.XX.X_to_develop</code> PR so it points to merge commit in gutenberg <code>trunk</code> for the gutenberg PR merged in the previous task.</p>
-<!-- /wp:paragraph -->
```

Comment:
I think we should keep updating the Gutenberg ref so it points to the merge commit, once the release PR on Gutenberg is merged into `trunk`. This way we would comply with the current pattern when merging a GB-mobile PR to point to the merge commit when changes are incorporated in the Gutenberg repo, wdyt?

Reply:
> Good catch. I added the Mobile App - Automation label to when the Gutenberg PR is created in the release automation script here https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/commit/7082bd81945b8760a78c9bbe6e9f7f38a55465a5

The `Mobile App - Automation` label is used mainly for generating automatically the companion Gutenberg Mobile PR ([example](https://github.com/wordpress-mobile/gutenberg-mobile/pull/4589)), since the release script is already creating that PR ([reference](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/67b902138f598d7e68caf8f62e443360e4e0a03a/release_automation.sh#L240-L257)), maybe this change would cause to generate a duplicated PR. I'm not sure about the benefit of using the `Mobile App - Automation` label if the script is already in charge of creating the PRs, wdyt?

Reply:
Yup, I understand what the `Mobile App - Automation` does and why it's used.

I suppose I got tripped up on the request to update the Gutenberg ref at this location in the release steps. 

At this point the release script generated Gutenberg Mobile PR [reference](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/67b902138f598d7e68caf8f62e443360e4e0a03a/release_automation.sh#L240-L257) has been merged into trunk [reference](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blame/develop/Releasing.md#L174) 

We would need an additional PR to merge hence the use of the automated companion PR.

But taking a step back, it seems we don't need this "Sync the Release to Development Branches" section at all.
Right now in the "Integrate the Release" section we have:

1. Verify that gutenberg ref in the GBM release PR is pointing to the HEAD of the GB release PR.
2. Tag HEAD of GB release PR with the release version
3. re-bundle and push any changes to GBM release PR
4.  Merge GBM release PR
5. Create a new GBM release from trunk at the release tag
6. Continue with main app steps

Should now be

1. Tag and merge GB release PR 
2. Update gutenberg ref and re-bundle  in the GBM release PR
3. Create a new GBM release from the release branch 
4. Merge GBM release PR
5. Continue with main apps

Reply:
I was imagining a slightly different process:

1. Tagging and releasing
    1. Tag GB release PR
    2. Tag GB-Mobile release PR and generate release (after verifying it is pointing at the GB release commit)
    3. Could continue with the main apps here.
2. Merging (this is basically the "Sync the Release to Development Branches" phase)
    1. Merge GB PR
    2. Update GB-Mobile PR to point at merge commit
    3. Merge GB-Mobile PR.
  
This feels more straightforward to me because of how the process of tagging/releasing GB/GB-mobile is separated from the process of merging them back to the `trunk` branches, but wdyt?

Reply:
For me it feels like context switching to move off the GB related repos to work on the main apps steps and relevant notifications. Then move back to the finishing GB steps. 
Although I can see that resolving a merge conflict would have less of a priority than finishing the release on the main apps and so forth. Also, separating out the merge steps will make it easier to experiment with forgoing the merge steps all together, if we opt to follow the Gutenberg release patterns.

Reply:
@fluiddot and @mchowning I _believe_ this is resolved in [7908aef](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/pull/95/commits/7908aefcbe2539d69b7b01910a8d78d3de72121e) 
Thanks for the feedback!